docs/components/collapse.md

```diff
@@ -0,0 +1,186 @@
+# Collapse
+
+**KCollapse** - A collapsible section.
+
+<KCollapse>
+  Can you see me now?
+</KCollapse>
+
+```html
+<KCollapse>
+  Can you see me now?
+</KCollapse>
+```
+
+## Props
+
+### title
+
+Use this prop to apply a title to the collapsible section.
+
+<KCollapse title="Look Mah!">
+  Can you see me now?
+</KCollapse>
+
+```html
+<KCollapse title="Look Mah!">
+  Can you see me now?
+</KCollapse>
+```
+
+### triggerLabel
+
+Use this prop to customize the content to trigger the collapse. The label will be displayed to the right of a caret that indicates the collapsedness.
+If no label is provided, only a caret will be displayed.
+
+<KCollapse trigger-label="View more info">
+  Can you see me now?
+</KCollapse>
+
+```html
+<KCollapse trigger-label="View more info">
+  Can you see me now?
+</KCollapse>
+```
+
+### triggerAlignment
+
+You can customize the alignment of the trigger. `leading` or `trailing` (default) alignment is supported.
+If a `title` is specified, the trigger will be inline with `trailing` alignment, or displayed beneath the `title` with `leading` alignment.
+
+<KCollapse trigger-label="View more info" trigger-alignment="leading">
+  Can you see me now?
+</KCollapse>
+
+<br />
+
+<KCollapse title="Look Mah!" trigger-label="What?" trigger-alignment="leading">
+  Can you see me now?
+</KCollapse>
+
+```html
+<KCollapse trigger-label="View more info" trigger-alignment="leading">
+  Can you see me now?
+</KCollapse>
+
+<br />
+
+<KCollapse title="Look Mah!" trigger-label="What?" trigger-alignment="leading">
+  Can you see me now?
+</KCollapse>
+```
+
+### v-model
+
+KCollapse can be controlled with `v-model`.
+
+<div class="d-flex mb-3">
+  <KLabel>Is Collapsed:</KLabel> {{ myIsCollapsed }}
+  <KButton class="ml-auto" @click="myIsCollapsed = !myIsCollapsed">Another trigger</KButton>
+</div>
+<div>
+  <KCollapse v-model="myIsCollapsed">
+    Can you see me now?
+  </KCollapse>
+</div>
+
+```html
+<div class="d-flex mb-3">
+  <KLabel>Is Collapsed:</KLabel> {{ myIsCollapsed }}
+  <KButton class="ml-auto" @click="myIsCollapsed = !myIsCollapsed">Another trigger</KButton>
+</div>
+<div>
+  <KCollapse v-model="myIsCollapsed">
+    Can you see me now?
+  </KCollapse>
+</div>
+```
+
+### modelValue
+
+To set the default collapse state without binding to v-model you can use `modelValue`.
+
+<KCollapse :model-value="false">
+  I am expanded by default
+</KCollapse>
+
+```html
+<KCollapse :model-value="false">
+  I am expanded by default
+</KCollapse>
+```
+
+## Slots
+
+- `default` - Content to be hidden or shown when clicking the trigger
+- `visible-content` - Content displayed above the collapsible content that is always visible
+- `trigger-content` - Contents of the trigger
+- `trigger` - Completely control the trigger, including managing click events
```

Comment:
Need more explanation on trigger vs. trigger-content

Reply:
How is this?

![image](https://user-images.githubusercontent.com/67973710/195620675-8590fd9a-6f0a-40f2-b5bb-54c530b99518.png)